steps/app-center-apk-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,65 @@
+title: App Center Apk Upload
+summary: |
+  Uploads an Android APK to App Center
+description: |
+  Uploads an Android APK to App Center and deploys to a given distribution group
+website: https://github.com/Ian-M-Cowley/bitrise-step-appcenterapkupload
+source_code_url: https://github.com/Ian-M-Cowley/bitrise-step-appcenterapkupload
+support_url: https://github.com/Ian-M-Cowley/bitrise-step-appcenterapkupload/issues
+published_at: 2018-12-11T14:35:31.09915726-06:00
+source:
+  git: https://github.com/Ian-M-Cowley/bitrise-step-appcenterapkupload.git
+  commit: f115f9cc1dd230558bca1db23e4ba7a462768c95
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: npm
+  apt_get:
+  - name: npm
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- app_path: $BITRISE_APK_PATH
+  opts:
+    description: |
+      App file path, for example to an APK file for Android.
+    is_expand: true
+    is_required: true
+    summary: ""
+    title: App path
+- app_center_app: $APP_CENTER_APP
```

Comment:
You should reference environment variables only if you can be sure they have a value, for example, when it is a variable exposed by Bitrise (as in the case of $BITRISE_APK_PATH) or if it is an output of a previous step.

In this setup, the user will see "$APP_CENTER_APP" as the value for the required input and could assume all is well, when in fact, running the step will fail with an error.

Please remove this environment variable reference and the similar ones following below and I'll be happy to merge your PR. :)